pkg/convert/testdata/programs/required_providers/main.tf

```diff
@@ -0,0 +1,16 @@
+terraform {
+  required_providers {
+    random = {
+      version = "~> 3.5.1"
+      source  = "hashicorp/random"
+    }
+    null = {
```

Comment:
Add a case here where the name doesn't match the source, e.g. like in https://developer.hashicorp.com/terraform/language/providers/requirements#handling-local-name-conflicts

Reply:
done.